core/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/status/SnapshotsStatusRequest.java

```diff
@@ -112,17 +114,40 @@ public SnapshotsStatusRequest snapshots(String[] snapshots) {
         return this;
     }
 
+    /**
+     * Set to <code>true</code> to ignore unavailable snapshots, instead of throwing an exception.
+     * Defaults to <code>false</code>, which means unavailable snapshots cause an exception to be thrown.
+     *
+     * @param ignoreUnavailable whether to ignore unavailable snapshots
```

Comment:
I agree, I usually don't add them but in this case I did to be consistent with the rest of the class' javadocs